.github/workflows/publish.yml

```diff
@@ -26,6 +26,8 @@ jobs:
         run: |
           git fetch --unshallow
           sudo bash -c "echo '$GPG_KEY_CONTENTS' | base64 -d > '$SIGNING_SECRET_KEY_RING_FILE'"
+      - name: Make gradlew executable
+        run: chmod +x ./gradlew
```

Comment:
you shouldn't need to do this in the publish yaml AFAIK...it should be checked into the git repo w/ the executable flag set... 🤔